lib/setup.bash

```diff
@@ -26,15 +26,7 @@ function setup() {
 	local -r asset_id=$(get_asset_id "$version")
 	mkdir -p "${bin_dir}"
 	download_asset "$asset_id" "$bin_dir"
-
-	export PATH="${bin_dir}:${PATH}"
-	export GITHUB_TOKEN="${GH_TOKEN}"
-	export ELASTIC_APM_ENVIRONMENT="ci"
-
-	git config --global user.name "${GIT_USER:-"${username}"}"
```

Comment:
what's the reason for this change?

Reply:
Not sure. I am still trying to figure out what's best.

I got it working in https://github.com/elastic/apm-managed-service/pull/890

But I needed to run `gh auth setup-git`. And didn't want to add it here. Not sure.

Unless I just add `gh` as a requirement. 

But in the e2e pipeline of this repo `gh auth setup-git` is not needed.

Reply:
Also, we are not doing the setup-git step in bolt-actions aswell, there it's also a different action doing it before oblt-cli setup.

Reply:
Do you think we can rename `setup.bash` for `install.bash`?